config/src/main/java/com/typesafe/config/impl/Tokenizer.java

```diff
@@ -299,7 +299,7 @@ private Token pullComment(int firstChar) {
         // chars JSON allows to be part of a number
         static final String numberChars = "0123456789eE+-.";
         // chars that stop an unquoted string
-        static final String notInUnquotedText = "$\"{}[]:=,+#`^?!@*&\\";
+        static final String notInUnquotedText = "$\"{}()[]:=,+#`^?!@*&\\";
```

Comment:
I don't know offhand if parens were allowed in unquoted strings before, but if so they should continue to be allowed.